recipes/libnvjitlink/meta.yaml

```diff
@@ -0,0 +1,87 @@
+{% set name = "libnvjitlink" %}
+{% set version = "12.0.76" %}
+{% set platform = "linux-x86_64" %}  # [linux64]
+{% set platform = "linux-ppc64le" %}  # [ppc64le]
+{% set platform = "linux-sbsa" %}  # [aarch64]
+{% set platform = "windows-x86_64" %}  # [win]
+{% set extension = "tar.xz" %}  # [not win]
+{% set extension = "zip" %}  # [win]
+{% set major_ver = version.split(".")[0] %}  # also tracks the CUDA Toolkit major version
+
+package:
+  name: {{ name|lower }}
+  version: {{ version }}
+
+source:
+  url: https://developer.download.nvidia.com/compute/cuda/redist/{{ name }}/{{ platform }}/{{ name }}-{{ platform }}-{{ version }}-archive.{{ extension }}
+  sha256: 92a70db1ee90f1816a40d4d2019e0a59c43eebb9f3eb1208abedcc00a13e48be  # [linux64]
+  sha256: dd317e1d052b4231ab95d1b472105640f617f7d250aacc5bc1c952aab93d7483  # [ppc64le]
+  sha256: 9f18090f6587e95be0ee424f07358eeb0e15c5ab3f8636870f68dd84a92ed2e6  # [aarch64]
+  sha256: cae74b6790174b283d892bb20e1ff91bff13c8a2a8486de4f417e09675469372  # [win]
+
+build:
+  number: 0
+  skip: true  # [osx]
+
+requirements:
+  build:
+    - {{ compiler("c") }}
+    - {{ compiler("cxx") }}
+
+outputs:
+  - name: libnvjitlink
+    test:
+      commands:
+        - test -f $PREFIX/lib/linvjitlink.so.{{ major_ver }}  # [linux]
+        - test -f $PREFIX/targets/*/lib/linvjitlink.so.{{ version }}  # [linux]
+        - if not exist %LIBRARY_BIN%\nvJitLink*.dll exit 1            # [win]
+    files:
+      - lib/libnvJitLink.so.*            # [linux]
+      - targets/*/lib/libnvJitLink.so.*  # [linux]
+      - Library\bin                      # [win]
+
+  - name: libnvjitlink-dev
+    run_exports:
+      - {{ pin_compatible("libnvjitlink", max_pin="x") }} 
+    test:
+      commands:
+        - test -f $PREFIX/include/nvJitLink.h               # [linux]
+        - test -f $PREFIX/lib/linvjitlink.so                # [linux]
+        - test -f $PREFIX/lib/linvjitlink.so.{{ version }}  # [linux]
+        - if not exist %LIBRARY_LIB%\nvJitLink.lib exit 1   # [win]
+    requirements:
+      run:
+        - libnvjitlink >={{ version }}
+        - libnvjitlink-static >={{ version }}
```

Comment:
```suggestion
    requirements:
      host:
        - cuda-version {{ cuda_version }}
      run:
        - {{ pin_compatible("cuda-version", max_pin="x.x") }}
        - libnvjitlink >={{ version }}
        - libnvjitlink-static >={{ version }}
```

Reply:
Q: Do we want to add `-static` as a requirement for `-dev`? We don't do that for other two packages above.

Reply:
We can remove the `-static` requirement here as well. Only `cuda-cudart-dev` is keeping it for nvcc support.

Reply:
We've been adding to `run_constrained` following this discussion ( https://github.com/conda-forge/staged-recipes/pull/21902#discussion_r1092460458 )

Though following this discussion sounds like we want an exact pin for the static library ( https://github.com/conda-forge/staged-recipes/pull/21901#discussion_r1145407909 )